stdlib/number.gr

```diff
@@ -398,3 +398,77 @@ export let sin = (radians: Number) => {
 export let cos = (radians: Number) => {
   sin(pi / 2 + radians)
 }
+
+/**
+ * Computes the tan of a number (in radians) using Chebyshev polynomials.
+ *
+ * @param radians: The input in radians
+ * @returns The computed tangent
+ *
+ * @since v0.5.2
+ */
+export let tan = (radians: Number) => {
+  sin(radians) / cos(radians)
+}
+
+/**
+ * Computes the factorial of a number.
+ *
+ * @param input: The number to compute the factorial of
+ * @returns The computed factorial
+ *
+ * @since v0.5.2
+ */
+export let factorial = input => {
```

Comment:
Missed opportunity for a recursive implementation! That said, this should fail if the import is not an integer, which makes it glaringly obvious that we're missing functions like `Number.isInteger`, `Number.isFloat`, and `Number.isRational`. https://github.com/grain-lang/grain/issues/1392

You could wait until those are implemented or remove `factorial` from this PR and bring it back later.